test/integration/integration_test.go

```diff
@@ -289,18 +329,37 @@ func commandReturnShouldBeEmpty(commandField string) error {
 	return compareExpectedWithActualEquals("", selectFieldFromLastOutput(commandField))
 }
 
-func executingMinishiftCommandSucceedsOrFails(command, expectedResult string) error {
-	err := minishift.executingMinishiftCommand(command)
+func commandReturnShouldNotBeEmpty(commandField string) error {
```

Comment:
I aim for re-use and short functions... but often it takes some time to reach this, and especially naming of functions is very important for this.